mixin/thanos/alerts/rule.libsonnet

```diff
@@ -171,15 +171,15 @@
             },
           },
           {
-            // NOTE: This alert will give false positive if no rules are configured.
-            alert: 'ThanosRuleTSDBNotIngestingSamples',
+            alert: 'ThanosNoRuleEvaluations',
             annotations: {
-              message: 'Thanos Rule {{$labels.job}} did not ingest any samples for the last 15 minutes.',
+              message: 'Thanos Rule {{$labels.job}} did not perform any rule evaluations in the past 2 minutes.',
             },
             expr: |||
-              sum by (job) (rate(prometheus_tsdb_head_samples_appended_total{%(selector)s}[5m])) <= 0
+              sum(rate(prometheus_rule_evaluations_total{%(selector)s}}[2m])) <= 0
```

Comment:
Is this setting an expectation that users are only doing rule evaluations at an interval of < 2 minutes? 

Do we instead want a variable that defaults to 2m in the range selector? 

I can't immediately think of an example of a rule that's evaluated over such infrequent intervals, but that may be a case for some consumers. Wdyt?

Reply:
The evaluation interval is for a rule group, so the expectation would be that even if the evaluation interval is 2m, at least 1 rule would have been attempted every 2 mins. And yes same as with scrapes, evaluation intervals higher than 2m don't really make sense. I'd prefer to not make things configurable prematurely.

Reply:
Sounds good, thanks for clarifying :)